### PR TITLE
Revision 2 - 28Nov22

### PR DESCRIPTION
**Pull request by craiglagegit**
Hi Patrick.  You asked that I create a pull request when pushing changes to a technote, so here it is.  I think I've addressed all of your comments from before, plus I've added data from two more recent fan speed tests.  It's a bit of a "data dump", but I'm not sure what else to do, really.

I'm not sure how you review the branch.  I guess you do a git pull and then build it locally.

### CODE REVIEW

File: index.rst [hunk 1]
```diff
@@ -15,14 +15,39 @@ Abstract
...
...installed in the AuxTel dome to aid with bringing the dome into temperature equilibrium with the outside air.  The fan is installed in one of the vent windows on the first floor and blows outward, drawing air into the dome through other openings (other vent windows, the dome slit).  The fan is a 3-phase helicoidal ventilator from Sodeca at https://sodeca.cl . It is model HC-100-6T/H IE3, and at 50Hz frequency it turns at 950 RPM and moves air at 37,000m^3/hr.  It has a variable frequency controller to control the fan speed from 10 Hz to 50 Hz.  There is a small control box to adjust the variable frequency.  When I ran the first two tests below, I made markings on the control box at about 25%, 50%, 75% and 100%, as seen in the image below.  After these tests, a frequency meter was installed to read the frequency (or maybe I just didn't see it the first time).  So on subsequent tests, I used the frequency as the measure of fan speed.  The table below shows the fan speed and air movement at the different settings.  Assuming the fan is exhausting a given volume of air and all of this air is entering through the slit, I made an estimate in the table of the wind speed coming in the slit.
```
Comment (patrickingraham) [inline index.rst:21]: Did you check that your markings are close to the actual powers?
Meaning, is your 50% marking pretty close to the 50% value on the VFD? Knobs are often non-linear..
if they're way off we should rub them out is all

File: index.rst [hunk 2]
```diff
@@ -15,14 +15,39 @@ Abstract
...
+
+.. figure:: ./_static/Fan.jpg
+    :width: 60%
+    :align: center 
+
+Fan mounted in vent gate window.
```
Comment (patrickingraham) [inline index.rst:27]: If you add a tab here, it'll make this sentence a caption for the figure.

Comment (patrickingraham) [reply index.rst:27]: Just for information sake, figures are generally better to use than images as they have more flexibility and features.
See here for info on how to use them.
https://docutils.sourceforge.io/docs/ref/rst/directives.html#figure
You can also add a reference to them, that can then be cross referenced.
https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#cross-referencing-arbitrary-locations

File: index.rst [hunk 3]
```diff
@@ -15,14 +15,39 @@ Abstract
...
+	    
+.. image:: ./_static/Control_box.jpg
+	:width: 45%
+	:align: right
+		
+.. image:: ./_static/Frequency_meter.jpg
+	:width: 45%
+	:align: left
+
+.. image:: ./_static/Fan_table.png
```
Comment (patrickingraham) [inline index.rst:38]: Suggest making this a figure, then adding a caption, especially if you refer to it later.

File: index.rst [hunk 4]
```diff
@@ -15,14 +15,39 @@ Abstract
...
+
+First test 22-Feb-22
+==================================
+
+After the fan was installed, it was noticed that it was causing degradation to the image quality and mount jitter.  So a series of tests were run to determine the cause.  One hypothesis was that the fan was causing vibration of the building, which was being transmitted to the telescope mount.  To test this, the fan was ramped up in speed while the other vent windows and the dome slit were closed.  The plots below show that in this condition, the fan speed has no impact on the mount jitter.  So this rules out vibration of the building as the cause of the tracking errors.  Also note that the azimuth oscillation seen in these tests was later understood to be due to a phenomenon called "cogging", which is unrelated to the fan-induced vibrations.
 
 .. image:: ./_static/Mount_Errors_Fan_22Feb22.png
 
-**Fan speed has no impact on mount tracking errors with slit closed.**
+**Fan speed has no impact on moount jitter with slit closed.**
```
Comment (patrickingraham) [inline index.rst:50]: moount :)

I'm a little confused as to what the bold faced text is used for.
Emphasizing the lesson learned is a good idea.  Maybe consider adding a prefix ?
Something like:
`Test result: Fan speed has no impact on mount jitter when the slit is closed`

Comment (patrickingraham) [reply index.rst:50]: Oh, i just realized that you are using these as captions??

If that's the case then switch these to figures, indent this line, remove the asterisks and you get captions!

File: index.rst [hunk 5]
```diff
@@ -31,7 +56,7 @@ A ventilation fan was installed in the AuxTel dome to aid with bringing the dome
 Second test 08-Mar-22
 =================================
 
-This test was then repeated with the dome slit open.  The plots below show the mount tracking error increasing steadily as the fan speed is increased.  The conclusion is that the mount tracking errors are caused by turbulent wind buffeting the mount.  The elevation mount tracking errors are also very similar to that seen when mount jitter is seen when observing during times of high wind speed.
+This test was then repeated with the dome slit open.  The plots below show the mount jitter increasing steadily as the fan speed is increased.  The conclusion is that the mount jitter is caused by turbulent wind buffeting the mount.  The elevation mount jitter is also very similar to that seen when observing during times of high wind speed.
```
Comment (patrickingraham) [inline index.rst:59]: I'm a bit embarrassed to say that I don't really know what "wind buffeting" is. I thought it was "vibrations" from pressure changes on either side of a barrier. Is that what you mean? Or do you just mean general turbulence that shakes the telescope (e.g. wind shake)?

File: index.rst [hunk 6]
```diff
@@ -31,7 +56,7 @@ A ventilation fan was installed in the AuxTel dome to aid with bringing the dome
...
 
 Fan speed at 0%
```
Comment (patrickingraham) [inline index.rst:61]: I think you can make these headings into captions, which probably makes more sense?

File: index.rst [hunk 7]
```diff
@@ -221,5 +246,128 @@ Banging on the side of the strut to excite some kind of bending mode was less su
...
+
+The tests were repeated on 23-Nov-22.  The wind velocity outside the dome as measured by the Gemini weather station was 3.6 m/s from 302 degrees.  The slit was only about 50% open to prevent the sun from falling on the telescope.  The telescope and dome were aligned to 180 degrees.  The figure below shows the setup: 
+
+		
+.. image:: ./_static/Dome_Setup_23Nov22.jpg
+
+**Setup on 23-Nov-22**
+
+
+These images show the anemometer readings, the accelerometer readings, and the mount encoder data as the fan speed is ramped up:
```
Comment (patrickingraham) [inline index.rst:269]: Do you have an information about the axes of the anemometer and how they roughly relate to the el/az?
Naively I would expect some sort of flow/turbulence in the elevation direction, which would correspond to one of the anemometer axes?

File: index.rst [hunk 8]
```diff
@@ -221,5 +246,128 @@ Banging on the side of the strut to excite some kind of bending mode was less su
...
+		
+.. image:: ./_static/Dome_Setup_28Nov22.jpg
+
+**Setup on 28-Nov-22**
+
+
+These images show the anemometer readings, the accelerometer readings, and the mount encoder data as the fan speed is ramped up:
+
+
+Measurements with Fan Speed at 0.0 Hz
```
Comment (patrickingraham) [inline index.rst:326]: boatload of power at 70 Hz in AuxTel-Truss Opt... but then seems to die out? any idea what that is?

File: index.rst [hunk 9]
```diff
@@ -221,5 +246,128 @@ Banging on the side of the strut to excite some kind of bending mode was less su
...
+
+
+Conclusions
+=================================
+
+This report is somewhat of a "data dump", with a lot of data that is difficult to interpret.  One consistent finding in all of the tests is that the mount jitter increases steadily as the fan speed is increased.  For this reason, it is not recommended to go above 20 Hz on the frequency meter when observing.  We should discontinue use of the hand written (25%-50%-75%) markings on the speed adjustment and use instead the frequency meter, which is much more accurate.  The accelerometer and anemometer readings are qualitatively consistent with increasing wind turbulence and mount shake as the wind speed is increased, but it is difficult to draw any quantitative conclusions from these readings. One clear result is that the anemometer standard deviations increase consistently as the fan speed is increased, indicating increased turbulence.  I am open to suggestions as to further measurements or analysis.
```
Comment (patrickingraham) [inline index.rst:370]: D'Oh you answered my question above about the hand written parts here...

**Review by patrickingraham — CHANGES_REQUESTED** [review]
Thanks Craig. Nice to see this coming together, even if we don't understand a bunch of it.
I made some comments about the use of figures that I think would help the readability of much of this.

For the future, it's useful to use [semantic line breaks](https://sembr.org/) when writing digital documentation. The main reason is that when you edit a single word in a paragraph, it will mark only that line (which is normally a sentence) as being changed and not the entire paragraph. I'm not suggesting you go back and change the entire document! I often just do bits at a time when I'm making small edits and chip away at it.

Lastly, we should replace the zip ties with metal hose clamps. Zip ties fail all the time...

File: index.rst (hunk 9 above)
Comment (craiglagegit) [reply index.rst:370]: I think I've made all of the requested changes.  Merging.

**Pull request merged**